packages/email-auth/src/email-auth-server.ts

```diff
@@ -17,118 +16,124 @@ export class EmailAuthServer {
     },
   ) {}
 
-  async init() {
-    this.server.on('request', async (req, res) => {
-      const url = new URL(
-        req.url!,
-        `${!req.headers.host!.startsWith('https://') ? 'https://' : ''}${
-          req.headers.host
-        }`,
-      );
-
-      if (
-        this.opts?.urlPrefix &&
-        !url.pathname.startsWith(this.opts?.urlPrefix)
-      ) {
-        return;
-      }
-
-      if (
-        req.method === 'POST' &&
-        `${this.opts?.urlPrefix ? this.opts?.urlPrefix : ''}/otp/generate` ===
-          url.pathname
-      ) {
-        const body = await parse.json(req);
-
-        const user = await this.storageAdapter.findOneUserByEmail({
-          emailAddress: body.emailAddress,
+  async expressMiddleware(
```

Comment:
this is not right,
this method should return a middleware function

Reply:
```
public expressMiddleware() {
  const { storageAdapter, emailAdapter } = this;
  return function(req: Request, res: Response, next: NextFunction) {
  
  }
}
```

Reply:
fixed